### PR TITLE
Update link

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

**Comment by mtamboli** [comment]
@dmuelle Everything looks good. Found a small typo `Then log with the admin username and password.` I think it should be `Then login with the admin username and password.` Would it be easier to make this change for you before we approve?

**Comment by dmuelle** [comment]
@mtamboli good catch- I fixed this. But in looking at that section, the instructions to access the UI seemed a little unclear. I tried to reformat a bit to separate out the Red Hat vs kubectl options. Let me know what you think.

File: posts/2024-04-26-ArgoCD-Configuration-Drift.adoc [hunk 1]
```diff
@@ -71,7 +71,16 @@ To get the default Admin password, run the following command:
 kubectl get secret argocd-initial-admin-secret -n argocd -o jsonpath='{.data.password}' | base64 -d
```
Comment (dmuelle) [inline posts/2024-04-26-ArgoCD-Configuration-Drift.adoc:70]: does this command return the default admin password as console output that the user copies and enters on the UI login page?

Comment (mtamboli) [reply posts/2024-04-26-ArgoCD-Configuration-Drift.adoc:70]: Yes, it output the password but you need to paste it to console.

Comment (dmuelle) [reply posts/2024-04-26-ArgoCD-Configuration-Drift.adoc:70]: ok- I revised this section slightly, just need to make sure I've understood correctly. You can review the update on the draft site:

https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2024/04/26/ArgoCD-Configuration-Drift.html#setup_argocd

**Review by mtamboli — APPROVED** [review]
It looks good to me. Thanks!

**Pull request merged**